tests/installation/first_boot.pm

```diff
@@ -85,6 +85,11 @@ sub test_flags() {
 sub post_fail_hook() {
     my $self = shift;
 
+    # Core dump for bsc#1005313
```

Comment:
thank you, that should make it a bit more obvious why much more space is used :-)